docs/basic.rst

```diff
@@ -183,25 +183,28 @@ Slot Status and Skipping Slot Updates
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 RAUC hashes each image or archive when packing it into a bundle and stores this
```

Comment:
Hmm, in reality it *is* a hash, so I'm not sure that this change actually makes it less confusing. As we can't easily change the name in the manifest, perhaps we should add more details here?
```suggestion
RAUC hashes each image or archive with SHA 256 when packing it into a bundle and stores this
```

Reply:
Added